hub/saml-engine/src/main/java/uk/gov/ida/hub/samlengine/SamlEngineModule.java

```diff
@@ -284,7 +284,7 @@ public PassthroughAssertionUnmarshaller getPassthroughAssertionUnmarshaller() {
 
     @Provides
     public IdpIdaStatusUnmarshaller getIdpIdaStatusUnmarshaller() {
-        return new IdpIdaStatusUnmarshaller(new IdpIdaStatus.IdpIdaStatusFactory(), new SamlStatusToIdpIdaStatusMappingsFactory());
+        return new IdpIdaStatusUnmarshaller();
```

Comment:
Doesn't need to be done as this PR...

I don't think there's a need have this in the binding module. We can just get whatever needs it to new one up.

Reply:
OK we'll remove this in #151

Reply:
🙆